stripe/src/main/java/com/stripe/android/paymentsheet/ui/BaseSheetActivity.kt

```diff
@@ -60,6 +62,17 @@ internal abstract class BaseSheetActivity<ResultType> : AppCompatActivity() {
             }
         }
 
+        bottomSheet.layoutTransition.enableTransitionType(LayoutTransition.CHANGING)
```

Comment:
unit tests?

Reply:
Added a check that this transition type is enabled.